MetaMask/metamask-extension
# Fix #19371 - Provide conversion buttons for empty accounts

## **Description**
Provides conversion buttons to buy, receive, and learn more about NFTs when an account is empty.

## **Manual testing steps**
- Start with a MetaMask account with assets in it on Mainnet
- *Don't* see the Buy/Receive blocks
- Create a new account with no assets
- *See* the Buy/Receive blocks
- Switch to a network that we don't support buying on (localhost)
- See *only* the "Receive" block
- Switch to NFTs tab
- See the NFTs block if you have no NFTS
- *Don't* see the NFTs block if you *do* have NFTs

## **Screenshots/Recordings**

### Token List
<img width="799" alt="SCR-20230928-qwwf" src="https://github.com/MetaMask/metamask-extension/assets/46655/ac72c151-ea8f-479d-a86a-e57d51164ccb">

### NFTs
<img width="815" alt="SCR-20230928-qxau" src="https://github.com/MetaMask/metamask-extension/assets/46655/910a8d69-3ff5-47ae-9b0a-47b9f10739e1">


## **Related issues**

_Fixes https://github.com/MetaMask/metamask-extension/issues/19371

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
